test/e2e/run_restart_test.go

```diff
@@ -35,11 +33,12 @@ var _ = Describe("Podman run restart containers", func() {
 	})
 
 	It("Podman start after successful run", func() {
-		session := podmanTest.Podman([]string{"run", ALPINE, "ls"})
+		SkipIfRemote()
+		session := podmanTest.Podman([]string{"run", "--name", "test", ALPINE, "ls"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
-		session2 := podmanTest.Podman([]string{"start", "--attach", "--latest"})
+		session2 := podmanTest.Podman([]string{"start", "--attach", "test"})
```

Comment:
another latest question

Reply:
Sure if we want to have one test to test --latest option, but I believe there is more value in testing these tests as remote.